lib/content-services/content-node-selector/content-node-selector.component.scss

```diff
@@ -1,6 +1,5 @@
 @mixin adf-content-node-selector-theme($theme) {
```

Comment:
all this style is wrong should not refer directly to the colours otherwise is not compatible with the theming

Reply:
I've changed the colors as you requested, please review. Thank you.